tests/sku-init/sku-init.test.ts

```diff
@@ -44,6 +44,7 @@ describe('sku init', () => {
 
   afterAll(async () => {
     await fs.rm(projectDirectory, { recursive: true, force: true });
+    spawnSync('git', ['restore', 'pnpm-lock.yaml']);
```

Comment:
I think in the past just running `pnpm install` after deleting the directory created by `sku init` was enough to restore the lockfile to its previous state, but this doesn't occur consistently, so I've opted to restore the lockfile before running `pnpm install`.

Reply:
The downside of this is that it will destroy any lockfile changes you have in your working tree. Maybe we store the lockfile, run the tests, then write it back, just to ensure nothing changes?

Reply:
Implemented in https://github.com/seek-oss/sku/pull/1284/commits/b63bca875cf025bfe5d1ef2c927a96006e520cc3